opty/direct_collocation.py

```diff
@@ -133,10 +133,16 @@ class ConstraintCollocator(object):
     """
 
     time_interval_symbol = sym.Symbol('h', real=True)
+    expeirment_data_symbol = sym.Symbol('e', real=True)
```

Comment:
I am working on Ubuntu 14.04. I am struggling in the online editing, please
recommend me a text editor/IDE. Thanks.

On Thu, May 19, 2016 at 2:31 PM, Jason K. Moore <notifications@github.com>
wrote:

> In opty/direct_collocation.py
> <https://github.com/csu-hmc/opty/pull/40#discussion_r63930745>:
>
> > @@ -133,10 +133,16 @@ class ConstraintCollocator(object):
> >      """
> >
> >      time_interval_symbol = sym.Symbol('h', real=True)
> > +    expeirment_data_symbol = sym.Symbol('e', real=True)
>
> What text editor/IDE do you use? You should find one that checks all this
> stuff for you automatically. I can give recommendations if you tell me what
> OS you are working on.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/csu-hmc/opty/pull/40/files/2a129a976a12bdb95e64d7afaecd93fcc78afcaf#r63930745>
>